requirements.txt

```diff
@@ -1,6 +1,5 @@
 ocf_datapipes==3.2.*
-pvnet==2.6.*
-ocf_ml_metrics==0.0.*
+pvnet==3.0.*
```

Comment:
worth pinning a bit more? or maybe not. Perhaps fine like this, but in pvnet_app could be fully pinned